### ISSUE TITLE
Rename package?

### ISSUE THREAD

**Issue by lungd** [lungd/LTC.jl#101]

**Comment by lungd** [comment]
Split into multiple packages?
MTKLayer and Optimization
Predefined and extendable systems (#106)
Examples

**Comment by Vaibhavdixit02** [comment]
I agree with changing the name, splitting doesn't seem like a good idea since it's not that big and most things interact with each other

**Comment by lungd** [comment]
Any suggestions for a new name?
